torchvision/prototype/features/_feature.py

```diff
@@ -1,11 +1,14 @@
 from __future__ import annotations
 
-from typing import Any, Callable, cast, List, Mapping, Optional, Sequence, Tuple, Type, TypeVar, Union
+from typing import Any, Callable, cast, List, Mapping, Optional, Sequence, Tuple, Type, TYPE_CHECKING, TypeVar, Union
 
 import torch
 from torch._C import _TensorBase, DisableTorchFunction
 from torchvision.transforms import InterpolationMode
 
+if TYPE_CHECKING:
+    from ._image import ColorSpace
```

Comment:
We need this to avoid cyclic imports here. A better way would be to put all enums in a separate module from all others. For an example see #6398.